a5.cpp

```
@@ -0,0 +1,8 @@
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
+    int v=10;
+    int ans=v/0;
```

cppcheck (2.10-2)

[error] Division by zero.

(zerodiv)